bin/clover/src/pipeline-steps/assetSpecificOverrides.ts

```diff
@@ -935,6 +935,32 @@ const overrides = new Map<string, OverrideFn>([
     setAnnotationOnSocket(idSocket, { tokens: ["TransitGatewayId"] });
     setAnnotationOnSocket(idSocket, { tokens: ["Transit Gateway Id"] });
   }],
+  ["AWS::EC2::VPNConnection", (spec: ExpandedPkgSpec) => {
+    const variant = spec.schemas[0].variants[0];
+    const domain = variant.domain;
+
+    const transitGatewayAttachmentIdProp = createScalarProp(
+      "TransitGatewayAttachmentId",
+      "string",
+      ["root", "domain"],
```

Comment:
i think this will be a resource prop as the data will be set from the AWS response - not via anything the user will set